rules/integrations/aws/defense_evasion_cloudtrail_logging_deleted.toml

```diff
@@ -20,7 +20,64 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS CloudTrail Log Deleted"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating AWS CloudTrail Log Deleted
+
+Amazon CloudTrail is a service that enables governance, compliance, operational auditing, and risk auditing of your
+Amazon Web Services account. With CloudTrail, you can log, continuously monitor, and retain account activity related to
+actions across your Amazon Web Services infrastructure. CloudTrail provides event history of your Amazon Web Services
+account activity, including actions taken through the Amazon Management Console, Amazon SDKs, command line tools, and
+other Amazon Web Services services. This event history simplifies security analysis, resource change tracking, and
+troubleshooting.
+
+This rule identifies the deletion of an AWS log trail using the API `DeleteTrail` action. Attackers can do this to
+cover their tracks and impact security monitoring that relies on this source.
+
+#### Possible investigation steps
+
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Investigate other alerts associated with the user account during the past 48 hours.
+- Contact the account and resource owners and confirm whether they are aware of this activity.
+- Check if this operation was approved and performed according to the organization's change management policy.
+- Considering the source IP address and geolocation of the user who issued the command:
+    - Do they look normal for the user?
+    - If the source is an EC2 IP address, is it associated with an EC2 instance in one of your accounts or is the source
+    IP from an EC2 instance that's not under your control?
+    - If it is an authorized EC2 instance, is the activity associated with normal behavior for the instance role or roles?
+    Are there any other alerts or signs of suspicious activity involving this instance?
+- Investigate the deleted log trail's criticality and whether the responsible team is aware of the deletion.
+- If you suspect the account has been compromised, scope potentially compromised assets by tracking servers, services,
+and data accessed by the account in the last 24 hours.
+
+### False positive analysis
+
+- If this rule is noisy in your environment due to expected activity, consider adding exceptions — preferably with a
+combination of user and IP address conditions.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Disable or limit the account during the investigation and response.
+- Identify the possible impact of the incident and prioritize accordingly; the following actions can help you gain context:
+    - Identify the account role in the cloud environment.
+    - Assess the criticality of affected services and servers.
+    - Work with your IT team to identify the impact on users.
```

Comment:
```suggestion
    - Work with your IT team to identify and minimize the impact on users.
```

Reply:
Leaving this comment in case you wanted to convey [the same message](https://github.com/elastic/detection-rules/pull/2124/files#diff-f949dcb17bba01b649173e23893602cd1dee197cf4f92c2e8dda014fadf397e4R56) that you used in other investigation guides. If you did want to include this, it'd apply across all other instances in this PR.